src/pages/Markets/components/LpGrid.tsx

```diff
@@ -60,6 +64,15 @@ export const LpGrid: React.FC<{
     )
   }
 
+  if (!filteredAssetIds.length)
+    return (
+      <ResultsEmpty
+        title='lending.yourLoans.emptyTitle'
+        body='lending.yourLoans.emptyBody'
```

Comment:
we should use more generic defi/positions vernacular here. not even showing loans as it stands.

Reply:
Ohh nice catch. Didn't catch since we should *not* hit this 🧠 

03824c8561